ivre/db/__init__.py

```diff
@@ -271,11 +277,13 @@ def __init__(self):
         self.argparser.add_argument('--openport', action='store_true')
         self.argparser.add_argument('--no-openport', action='store_true')
         self.argparser.add_argument('--countports', metavar='COUNT',
-                                    help='show only results with a number of open '
-                                    'ports within the provided range', nargs=2)
+                                    help='show only results with a number '
+                                    'of open ports within the provided range',
+                                    nargs=2)
         self.argparser.add_argument('--no-countports', metavar='COUNT',
-                                    help='show only results with a number of open '
-                                    'ports NOT within the provided range', nargs=2)
+                                    help='show only results with a number '
```

Comment:
Please do not change files when it's not needed, as it makes `git blame` unnecessarily harder to use.